api/externals/repository/activity_repository.go

```diff
@@ -20,6 +20,7 @@ type ActivityRepository interface {
 	Update(context.Context, string, string, string, string, string) error
 	Destroy(context.Context, string) error
 	AllWithSponsor(context.Context) (*sql.Rows, error)
```

Comment:
```suggestion
	FindDetail(context.Context) (*sql.Rows, error)
```
activityに紐づくテーブルデータを取得する際のメソッドも修正してほしいです！
86行目のメソッド名もこれに合わせて`FindDetail`になります。